components/media_player/ui/CardMediaPlayerEPGView.qml

```diff
@@ -144,14 +144,15 @@ Rectangle {
                         height: item_height
                         width: item_width
                         border.color: Style.color.dark
-                        color: item_color
+                        color: item_epgItemColor
+                        
                         Text {
                             id: epgItemTitleText
                             text: item_title
                             elide: Text.ElideRight
                             width: epgItemRectangle.width
                             wrapMode: Text.NoWrap
-                            color: Style.color.dark
+                            color: item_epgItemTextColor
```

Comment:
Where is `item_epgItemTextColor` defined? I can't find it.

Reply:
in the integrations.ibrary epgmodel header file

Reply:
@martonborzak could you check this? To me as a non QML / JS developer I don't see the magic how this could work.
Nevertheless, if it's defined in the other integrations.library pull request, then we have to wait until it is merged & released, then update the dependency in this PR.

Reply:
It's defined in the QAbstractListmodel `roles[EpgItemColorRole] = "item_epgItemColor";`, so it should work, once the integrations.library is merged.

Reply:
That still doesn't answer my ignorance of QML since I only see a string of the same name but no property / object, but I trust you :-)

@martonborzak could you check the related PR YIO-Remote/integrations.library#15 ? Then I'll merge and create new releases.

Reply:
It's good.

The name of string is coming from how the listview handles the QAbstracListmodel. When you pass the model to the listview, it know all the properties automatically that was defined in c++.